api_spec/edge_api.json

```diff
@@ -178,6 +253,26 @@
                         "type": "string"
                     }
                 }
+            },
+            "BulkIdentitiesData": {
+                "required": true,
+                "type": "object",
+                "properties": {
+                    "data": {
+                        "$ref": "#/components/schemas/IdentitiesRequestData"
```

Comment:
This isn't quite right, the data attribute should be an array of IdentitiesRequestData objects.